src/Clay/Render.hs

```diff
@@ -184,7 +184,7 @@ mediaType (MediaType (Value v)) = fromText (plain v)
 feature :: Feature -> Builder
 feature (Feature k mv) =
   case mv of
-    Nothing        -> fromText k
+    Nothing        -> mconcat [ "(", fromText k, ")" ]
```

Comment:
I'm surprised, is this case ever tested?

Reply:
It would be with this change.  E.g., on line 52:

```haskell
"color" `shouldRenderFromFeature` color
```